substrate/client/src/client.rs

```diff
@@ -300,6 +300,11 @@ impl<B, E, Block> Client<B, E, Block> where
 		*self.importing_block.write() = Some(hash);
 		let result = self.execute_and_import_block(origin, hash, header, justification, body);
 		*self.importing_block.write() = None;
+		telemetry!("block.import";
+			"height" => { let n: u64 = header.number().as_(); n },
```

Comment:
shouldn't:
```
"height" => header.number().as_::<u64>(),
```
work?

Reply:
no - `as_` doesn't take a generic param: the trait does.